### PR TITLE
[LPTOCPCI-698] Fix MTR Scenario Failure in Scenario Setup

### PR DESCRIPTION
**Pull request by calebevans**

### CODE REVIEW

**Comment by calebevans** [comment]
/pj-rehearse periodic-ci-windup-windup-ui-tests-v1.1-mtr-ocp4.14-lp-interop-mtr-interop-aws

**Comment by calebevans** [comment]
/pj-rehearse periodic-ci-windup-windup-ui-tests-v1.1-mtr-ocp4.14-lp-interop-mtr-interop-aws

**Comment by calebevans** [comment]
/pj-rehearse periodic-ci-windup-windup-ui-tests-v1.1-mtr-ocp4.14-lp-interop-mtr-interop-aws

**Comment by calebevans** [comment]
/pj-rehearse periodic-ci-windup-windup-ui-tests-v1.1-mtr-ocp4.14-lp-interop-mtr-interop-aws

**Comment by calebevans** [comment]
/pj-rehearse periodic-ci-windup-windup-ui-tests-v1.1-mtr-ocp4.14-lp-interop-mtr-interop-aws

**Comment by calebevans** [comment]
/pj-rehearse periodic-ci-windup-windup-ui-tests-v1.1-mtr-ocp4.14-lp-interop-mtr-interop-aws

**Comment by calebevans** [comment]
/pj-rehearse periodic-ci-windup-windup-ui-tests-v1.1-mtr-ocp4.14-lp-interop-mtr-interop-aws

File: ci-operator/step-registry/mtr/deploy-windup/mtr-deploy-windup-commands.sh [hunk 1]
```diff
@@ -20,21 +20,7 @@ EOF
...
-        if [ $try == $RETRIES ]; then
-            echo "Error deploying Windup, exiting now"
-            exit 1
-        fi
-        echo "Try ${try}/${RETRIES}: Windup deployment is not ready. Checking again in 30 seconds"
-        sleep 30
-    fi
-done
\ No newline at end of file
+oc wait deployments -n mtr --all=true --for condition=Available --timeout=350s
```
Comment (ascerra) [inline ci-operator/step-registry/mtr/deploy-windup/mtr-deploy-windup-commands.sh:26]: This timeout seems really short compared to what was here before

we had 1800s (30m) before now this oc wait is only wait for 350s (roughly 5m)

**Pull request merged**